requirements-base.txt

```diff
@@ -54,7 +54,8 @@ requests>=2.25.1
 rfc3339-validator>=0.1.2
 rfc3986-validator>=0.1.1
 # [end] jsonschema format validators
-sentry-arroyo>=2.7.1
+sentry-arroyo[json]==2.7.1
```

Comment:
Just curious, any particular reason for changing it from `>=` to `==`?

Reply:
Copy/pasted it from somewhere else and didn't notice :) I can change it back